src/qs_moments.F

```diff
@@ -522,24 +519,14 @@ SUBROUTINE build_local_magmom_matrix(qs_env, magmom, nmoments, ref_point, ref_po
                              sphi_b(1, sgfb), SIZE(sphi_b, 1), &
                              0.0_dp, work(1, 1), SIZE(work, 1))
 
-                  IF (iatom <= jatom) THEN
+                  ! IF (iatom <= jatom) THEN
```

Comment:
please do not include commented out code, and re-indent the following code properly

Reply:
Done.